component-interaction/src/app/user-list/user-list.component.ts

```diff
@@ -0,0 +1,43 @@
+import { Component } from "@angular/core";
+
+import { UserModel } from "../user.model";
+
+@Component({
+	selector: "user-list",
+	templateUrl: "./user-list.component.html",
+	styleUrls: ["./user-list.component.css"]
+})
+
+export class UserListComponent {

```

Comment:
```suggestion
export class UserListComponent implements OnInit {

```
implement the interface